variables.tf

```diff
@@ -97,6 +97,13 @@ variable "member_disk_mb" {
   # Validation is done in the Terraform plan phase by the IBM provider, so no need to add extra validation here.
 }
 
+variable "member_host_flavor" {
+  type        = string
+  description = "Allocated host flavor per member. [Learn more](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/database#host_flavor)."
```

Comment:
Since it has a default value of `null`, we should include in the description what happens when no value is passed?

Reply:
The link points to the host flavor page. Is it best to let consumers read that page, ideally all new deployments would set a value to use the new model. null is only supported as a temporary measure until the assisted/forced migration is complete. My concern is that the behaviour is likely change over time. Today, the old hosting model is used, in the future it appears that the `Shared compute` model will be used.